include/openssl/macros.h

```diff
@@ -202,6 +202,16 @@
 #  define DEPRECATEDIN_0_9_8(f)      f;
 # endif
 
+/*
+ * Some deprecated functions are still needed for use internally within OpenSSL.
+ * Applications should *not* define the OPENSSL_INTERNAL_USE macro.
+ */
```

Comment:
And yet, we're using it in our app...  That usage kinda flies in the face of this comment...